Gopkg.toml

```diff
@@ -132,7 +132,6 @@
   name = "github.com/sirupsen/logrus"
   revision = "89742aefa4b206dcf400792f3bd35b542998eb3b"
 
-
 [[constraint]]
-  name = "github.com/ciao-project/ciao"
-  revision = "239f5bc3e8e5fe18e6b4df31dcd692db5692c55f"
+  branch = "master"
```

Comment:
Why did we switch the constraint to be just master?

Reply:
Oh yeah I didn't catch that, @devimc we want to stick with a specific revision of govmm here.

Reply:
ooops, fixed